service/src/main/java/uk/gov/hmcts/reform/fpl/model/event/PlacementEventData.java

```diff
@@ -73,9 +72,12 @@ public List<Element<Placement>> getPlacementsNonConfidential(boolean withNotices
         if (isEmpty(placements)) {
             return emptyList();
         }
+
         return placements.stream()
-            .map(element -> element(element.getId(), element.getValue().nonConfidential(withNoticesResponses)))
-            .collect(toList());
+                .filter(element -> element.getValue().getPlacementNotice() != null)
+                .filter(element -> !element.getValue().getPlacementNotice().isEmpty())
+                .map(element -> element(element.getId(), element.getValue().nonConfidential(withNoticesResponses)))
+                .toList();
```

Comment:
Just did a quick code search, looks like this method is invoked by ManageDoc event, migration service and the remove tools as well.
It should be fine for ManageDoc event, as user can only upload notice response when a notice exist.

But i am a bit concern about the removal tools. Will these changes disable admin from removing the placement application which don't have a notice yet? Or maybe potentially cause some leftover in the case data?

Reply:
If it won't break any other stuff, could we rename this method a bit to avoid any confusion in the future. Because, instead of getting non-confidential placement, this method is now getting non-confidential placement with notice.